src/views/main.js

```diff
@@ -66,6 +67,7 @@ const mainHeader = data => `
       </button>
     </li>
   </menu>
+  ${data.userMenu}
```

Comment:
To expand on my note above regarding having to pass user-menu data to every partial, I _think_ an answer might be to build the user menu into the main header.  Maybe this should be a pattern for persistent elements.  Still needs a think but I think we can improve our framework here.